migrator/migrations/n_01_to_n_02_postgres_clusters/migration.go

```diff
@@ -43,8 +44,13 @@ func move(gormDB *gorm.DB, postgresDB *pgxpool.Pool, legacyStore legacy.Store) e
 	ctx := sac.WithAllAccess(context.Background())
 	store := pgStore.New(postgresDB)
 	pkgSchema.ApplySchemaForTable(context.Background(), gormDB, schema.Table)
+	_, err := postgresDB.Exec(ctx, fmt.Sprintf("ALTER TABLE %s DISABLE TRIGGER ALL", schema.Table))
```

Comment:
No need to do this. Checkout `pghelper.ForT(s.T(), true)`
`		&gorm.Config{
			NamingStrategy:                           pgutils.NamingStrategy,
			DisableForeignKeyConstraintWhenMigrating: disableConstraint,
		},`